interfaces/cython/cantera/test/test_reaction.py

```diff
@@ -1722,7 +1717,7 @@ def setUp(self):
         super().setUp()
         self._rate = lambda T: 38.7 * T**2.7 * exp(-3150.15428/T)
 
-    def from_yaml(self, deprecated=False):
+    def from_yaml(self):
```

Comment:
Why is this test here?

Reply:
To override the implementation of `from_yaml` in the base `ReactionTests` class.

Reply:
There will be further changes necessary in subsequent PR's (for the removal of legacy `Kinetics`) - so this change is just an intermediate step.